Updates for new sensubility amqp1 address (#239)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_deploying-smart-gateways.adoc

= Deploying Smart Gateways

[role="_abstract"]
ifdef::include_when_16[]

> You must deploy a Smart Gateway for each of the data collection types for each cloud; one for collectd metrics, one for collectd events, one for Ceilometer metrics, one for Ceilometer events and one for collectd-sensubility metrics. Configure each of the Smart Gateways to listen on the AMQP address that you define for the corresponding cloud. Smart Gateways are defined via the `clouds` parameter in the `ServiceTelemetry` manifest.

Reviewer: ```suggestion
You must deploy a Smart Gateway for each of the data collection types for each cloud; one for collectd metrics, one for collectd events, one for Ceilometer metrics, one for Ceilometer events, and one for collectd-sensubility metrics. Configure each of the Smart Gateways to listen on the AMQP address that you define for the corresponding cloud. To  define Smart Gateways, you configure the `clouds` parameter in the `ServiceTelemetry` manifest.
```

endif::include_when_16[]
ifdef::include_when_13[]

> You must deploy a Smart Gateway for each of the data collection types for each cloud; one for collectd metrics, one for collectd events, one for Ceilometer metrics, one for Ceilometer events. Configure each of the Smart Gateways to listen on the AMQP address that you define for the corresponding cloud. Smart Gateways are defined via the `clouds` parameter in the `ServiceTelemetry` manifest.

Reviewer: ```suggestion
You must deploy a Smart Gateway for each of the data collection types for each cloud; one for collectd metrics, one for collectd events, one for Ceilometer metrics, and one for Ceilometer events. Configure each of the Smart Gateways to listen on the AMQP address that you define for the corresponding cloud. To define Smart Gateways, configure the `clouds` parameter in the `ServiceTelemetry` manifest.
```

Removed passive voice
